src/wallet/rpcwallet.cpp

```diff
@@ -709,8 +721,11 @@ static RPCHelpMan getreceivedbyaddress()
             + HelpExampleCli("getreceivedbyaddress", "\"" + EXAMPLE_ADDRESS[0] + "\" 0") +
             "\nThe amount with at least 6 confirmations\n"
             + HelpExampleCli("getreceivedbyaddress", "\"" + EXAMPLE_ADDRESS[0] + "\" 6") +
+            "\nThe amount with at least 6 confirmations including immature coinbase outputs\n"
+            + HelpExampleCli("getreceivedbyaddress", "\"" + EXAMPLE_ADDRESS[0] + "\" 6 true") +
             "\nAs a JSON-RPC call\n"
```

Comment:
It's odd that "As a JSON-RPC call" no longer refers to the example immediately above. I think only one JSON-RPC example is needed.